os/board/rtl8730e/Kconfig

```diff
@@ -65,6 +65,16 @@ config BOARD_FLASH_32M
 
 endchoice
 
+if SECOND_FLASH_PARTITION
+config SECOND_FLASH_START_ADDR
+    hex
+    default 0x09000000 if BOARD_FLASH_16M
+    default 0x0A000000 if BOARD_FLASH_32M
```

Comment:
This could give confusion.
How about renaming them like?
```
BOARD_FLASH_XX -> BOARD_PRI_FLASH_XX
SECOND_FLASH_XX -> BOARD_SEC_FLASH_XX
```
PRI is primary, SEC is secondary.
This can be done with new PR.

Reply:
Okay. I will update with new PR later.

Reply:
@jeongarmy Please find @Tirtha-Samsung 's comment on PR #6350 . He said, with latest downloader, we can't write something between 0x0900 0000 ~ 0x0A00 0000 regardless of the internal flash size.

Reply:
I updated always 0xA000000 regardless flash size.